src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -243,15 +200,22 @@
    (fn [{:keys [state]}]
      [:div {}
       [:div {:style {:backgroundColor "white" :paddingBottom "2em"}}
-       (when (:signed-in? @state) [LoggedIn {:nav-context (:root-nav-context @state)}])
-        ;; This has to be hidden rather than simply omitted. After a successful login, Google
-        ;; attempts to find the sign-in button and manipulate it, which throws an error if it is
-        ;; not present on the page.
-       [LoggedOut {:hidden? (:signed-in? @state)}]]
+       (if (:access-token @state)
+         [LoggedIn {:nav-context (:root-nav-context @state)}]
+         [LoggedOut])]
       (footer)])
    :component-did-mount
    (fn [{:keys [this state]}]
-     (.addEventListener js/window "hashchange" (partial react/call :handle-hash-change this)))})
+     (.addEventListener js/window "hashchange" (partial react/call :handle-hash-change this))
+     (let [hash (-> js/window .-location .-hash)
+           at-index (utils/str-index-of hash "?access_token=")
+           [_ token] (clojure.string/split (subs hash at-index) #"=")]
+       (when-not (neg? at-index)
+         (swap! state assoc :access-token token)
```

Comment:
So, this changes end-user behavior. Now, since we're only storing the access token in state, the user must re-login on every full browser reload - such as loading from a bookmark or opening a new tab.

I don't want to hold up this PR for this, but I do think we need to go back to the old behavior - maybe store the token in a cookie.